api/proto/banyandb/database/v1/rpc.proto

```diff
@@ -202,11 +230,33 @@ message MeasureRegistryServiceListResponse {
 }
 
 service MeasureRegistryService {
-  rpc Create(MeasureRegistryServiceCreateRequest) returns (MeasureRegistryServiceCreateResponse);
-  rpc Update(MeasureRegistryServiceUpdateRequest) returns (MeasureRegistryServiceUpdateResponse);
-  rpc Delete(MeasureRegistryServiceDeleteRequest) returns (MeasureRegistryServiceDeleteResponse);
-  rpc Get(MeasureRegistryServiceGetRequest) returns (MeasureRegistryServiceGetResponse);
-  rpc List(MeasureRegistryServiceListRequest) returns (MeasureRegistryServiceListResponse);
+  rpc Create(MeasureRegistryServiceCreateRequest) returns (MeasureRegistryServiceCreateResponse){
```

Comment:
CRUD path seems strange. They do not fully follow Rest style

Reply:
Could you be specific which part?

Reply:
> Could you be specific which part?

Comments added. But I am not sure if this is attributed to the limitation of `grpc-gateway`

Reply:
I will tweak them a bit